src/security-council-mgmt/governors/modules/SecurityCouncilNomineeElectionGovernorIndexingTiming.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../../interfaces/ISecurityCouncilManager.sol";
+
+import "@openzeppelin/contracts-upgradeable/governance/GovernorUpgradeable.sol";
+import "lib/solady/src/utils/DateTimeLib.sol";
+
+abstract contract SecurityCouncilNomineeElectionGovernorIndexingTiming is Initializable, GovernorUpgradeable {
```

Comment:
I think we should remove the indexing from this contract - the timing is nice to separate as it means the sub class doesnt have to deal with date manipulation.

Reply:
move these back?

```solidity
/// @notice Number of elections created
uint256 public electionCount;

.........

/// @notice Returns the cohort for a given `electionIndex`
function electionIndexToCohort(uint256 electionIndex) public pure returns (Cohort) {
    return Cohort(electionIndex % 2);
}

function cohortOfMostRecentElection() external view returns (Cohort) {
    return electionIndexToCohort(electionCount - 1);
}

/// @notice Returns the description for a given `electionIndex`
function electionIndexToDescription(uint256 electionIndex)
    public
    pure
    returns (string memory)
{
    return string.concat("Nominee Election #", StringsUpgradeable.toString(electionIndex));
}

/// @notice Returns the proposalId for a given `electionIndex`
function electionIndexToProposalId(uint256 electionIndex) public pure returns (uint256) {
    return hashProposal(
        new address[](1),
        new uint256[](1),
        new bytes[](1),
        keccak256(bytes(electionIndexToDescription(electionIndex)))
    );
}
```